pkg/client/client_test.go

```diff
@@ -2030,7 +2031,8 @@ U5wwSivyi7vmegHKmblOzNVKA5qPO8zWzqBC
 				Expect(actual).NotTo(BeNil())
 
 				By("validating the fetched Deployment equals the created one")
-				Expect(u).To(Equal(&actual))
+				unstructured.RemoveNestedField(actual.Object, "spec", "template", "metadata", "creationTimestamp")
```

Comment:
The two test changes are required because `creationTimestamp` is no omitted when null: https://github.com/kubernetes/kubernetes/pull/130989

Reply:
Any idea why the creationTimestamp still shows up with null in spec.template.metadtaa.creationTimestamp?

```json
 "spec" : {
    "replicas" : 2,
    "selector" : {
      "matchLabels" : {
        "foo" : "bar"
      }
    },
    "template" : {
      "metadata" : {
        "creationTimestamp" : null,
        "labels" : {
          "foo" : "bar"
        }
      },
...
```